README.md

```diff
@@ -28,19 +28,17 @@ Microsoft PowerToys is a set of utilities for power users to tune and streamline
 
 - Windows 11 or Windows 10 version 2004 (code name 20H1 / build number 19041) or newer.
 - Our installer will install the following items:
-   - [.NET 6.0.7 Desktop Runtime](https://dotnet.microsoft.com/download/dotnet/6.0#runtime-desktop-6.0.7) or a newer 6.0.x runtime.
+   - [.NET 6.0.7 Desktop Runtime](https://dotnet.microsoft.com/download/dotnet/6.0#runtime-desktop-6.0.7).
```

Comment:
It should work with a newer runtime still so this statement is correct.

Reply:
Okay, then I must have been confused by PRs like 19522 where I saw...
```wxs
<?define Dotnet6DownloadUrl="https://download.visualstudio.microsoft.com/download/pr/f33cf7ce-bf03-428c-8aa7-e32ef6d7ddc6/e61dc60fce686844c41ec2901ad5b01e/windowsdesktop-runtime-6.0.7-win-arm64.exe"?>
```
hardcoded version numbers.

Reply:
interesting, should be version detect of "or greater" but the wsx does want that very targeted version

Reply:
Does that mean my change is good?

Reply:
I think the language is correct, yes.